cdk/visit/lambda_code/quicksight/dashboard_generator.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import os
+
+def lambda_handler(event, context):

```

Comment:
And just so I understand, Cognito will allow the staff to authenticate and execute this lambda through the frontend to generate a link to view the dashboard? Other than the hardcoded arn/account above, everything is looking good so far!

Reply:
That's correct! The Cognito will be used for authentication to access both the quiz-status and the quicksight dashboard genertor.